src/redux/reducer.ts

```diff
@@ -347,19 +347,26 @@ export interface StudyUpdatedEventDataHeader {
 interface RootNetworkDeletionStartedEventDataHeader {
     studyUuid: UUID;
     rootNetworks: UUID[];
-    updateType: string;
+    updateType: NotificationType.ROOT_NETWORK_DELETION_STARTED;
 }
 
 interface LoadflowResultEventDataHeaders {
     studyUuid: UUID;
     rootNetwork: UUID; // todo rename rootNetworkUuid in back as well
-    updateType: string;
+    updateType: NotificationType.LOADFLOW_RESULT;
 }
 
 interface RootNetworkModifiedEventDataHeaders {
     studyUuid: UUID;
     rootNetwork: UUID; // todo rename rootNetworkUuid in back as well
-    updateType: string;
+    updateType: NotificationType.ROOT_NETWORK_MODIFIED;
```

Comment:
not right here, but in case an XXXEventData having several updateType, we should use union of all supported NotificationType for field _updateType_